repositories/rules.go

```diff
@@ -46,6 +46,14 @@ func (repo *MarbleDbRepository) ListRulesByIterationId(ctx context.Context, exec
 	)
 }
 
+func (repo *MarbleDbRepository) ListRulesExecutionByIterationId(ctx context.Context, exec Executor, iterationId string) ([]models.RuleExecutionStat, error) {
+	if err := validateMarbleDbExecutor(exec); err != nil {
+		return nil, err
+	}
+	// TO BE DEFINED
```

Comment:
Just a note, let's please do 2 separate queries this time to get the numbers for the "live" and "tested" versions rather than a union all, it's simpler to read and no longer to do for the db :-)